src/Dao/BikesDao.java

```diff
@@ -0,0 +1,71 @@
+package Dao;
+
+import Type.Bike;
+
+import java.io.*;
+import java.util.ArrayList;
+
+class BikesDao {
+
+    private ArrayList<Bike> bikes = new ArrayList<Bike>();
+    private File file = new File("BikeBase.txt");
+
+    void findAll() {
```

Comment:
it's not a `find all` because you not really find them, but mostly load.
So rename this method to "loadData"

Reply:
ok